nifi-commons/nifi-expression-language/src/test/java/org/apache/nifi/attribute/expression/language/TestQuery.java

```diff
@@ -1177,6 +1179,99 @@ public void testReplaceAllWithMatchingGroup() {
         verifyEquals("${attr:replaceAll('.*?(l+).*', '$1')}", attributes, "ll");
     }
 
+    @Test
+    public void testReplaceShouldReplaceAllLiteralMatches() {
+        int n = 3;
+        final String ORIGINAL_VALUE = "Hello World";
```

Comment:
Understanding that this is a migration, can you correct the style of these methods variables to use camelCase instead of all capital letters?
```suggestion
        final String originalValue = "Hello World";
```